docs/UserGuide.md

```diff
@@ -65,8 +65,8 @@ command to run the application.<br>
 * Items with `…`​ after them can be used multiple times including zero times.<br>
   e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend t/family` etc.
 
-* Items with `+`​ after them can be used multiple times, but requires at least one usage.<br>
-    e.g. `[t/TAG]+​` can be used as `t/friend`, `t/friend t/family` etc.
+* Items with `+` after them can be used multiple times, but requires at least one usage.<br>
+    e.g. `t/TAG+` can be used as `t/friend`, `t/friend t/family` etc.
```

Comment:
Should we standardised this to whether we want to use `[t/TAG]` or `t/TAG`? As line 66 shows` [t/TAG] `instead.

Reply:
This is under Features, so we are trying to explain to the user how to read our command. So the logic is that if t/tag have a square bracket, it should not have `+` because `+` means it is used at least once while [] means it is optional. So it is contradicting each other.